src/main/java/dk/frankbille/scoreboard/domain/User.java

```diff
@@ -26,6 +26,7 @@
 	private String username;
 	private Player player;
 	private League defaultLeague;
+	private RatingCalculatorType ratingCalculator;
```

Comment:
Hmm, I'm not so sure about the rating being controlled on user level. That would result in users not seeing the same scoreboard. I would think that it sits better on a League level. WDYT?